samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_Hirakund_reservoir_using_video.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p style="user-select: auto;">It turns out the font <code style="user-select: auto;">arial</code>  is named as <code style="user-select: auto;">Arial.ttf</code>  in Mac and this cell will throw an error if I did not change the case. Could you check the casing on Linux (perhaps Ubuntu) and add some try except block to try across a few different font file names?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Ahh, thanks for this review. I have put some if else conditions after checking the OS to handle this error in Windows, linux and MacOS.</p>